config/prod/cmc-migration-bucket.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.8.4/templates/S3/synapse-external-bucket.j2"
+stack_name: "cmc-migration-bucket"
+stack_tags:
+  OwnerEmail: 'thomas.yu@sagebase.org'
```

Comment:
I'm leaving Tom as owner of this bucket, let me know if you would prefer to change it to myself